cleric-quests.lic

```diff
@@ -138,11 +138,8 @@ class ClericQuests
     bput('meditate', 'You close your eyes and begin to meditate')
     waitfor('once again changing and reforming into your Human form')
 
-    case bput('dive deeper water', 'You take a deep breath', 'you find that you cannot move yourself to go in')
-    when 'you find that you cannot move yourself to go in'
-      echo("Pausing for 5 minutes until you can complete another ritual because you did tamsine's too recently.")
```

Comment:
❓ I liked this `echo`. Why was it removed?

Reply:
readded it, sorry :)